Cargo.toml

```diff
@@ -54,7 +58,7 @@ reqwest-native-tls-vendored = ["reqwest/native-tls-vendored"]
 ureq-rustls-tls = ["ureq/tls"]
 
 # Internal features for checking async or sync compilation
-__async = ["async-trait"]
+__async = ["async-trait", "async-stream", "futures"]
```

Comment:
Is `futures-util` crate necessary for `__async` feature, or we could remove `futures-util` from dependencies list and leave it in `dev-dependencies`?

Reply:
We only need it in `dev-dependencies`. Removed it from `dependencies`.